core/src/test/scala/org/apache/spark/util/JsonProtocolSuite.scala

```diff
@@ -1097,7 +1100,9 @@ private[spark] object JsonProtocolSuite extends Assertions {
       |  },
       |  "Task Metrics": {
       |    "Executor Deserialize Time": 300,
+      |    "Executor Deserialize CPU Time": 0,
```

Comment:
I still think it would be worth to fix this; just find some way around the style check.

Otherwise, you're not really testing whether the parsing code is actually parsing the field (what if there's a typo somewhere?).